alica_ros_proxy/src/clock/AlicaROSClock.cpp

```diff
@@ -1,23 +0,0 @@
-#include "clock/AlicaROSClock.h"
```

Comment:
We can still support this one. It is useful to connect with Ros' simulated time

Reply:
done